.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,87 @@
+---
+name: Bug Report
+about: Report a problem with qBittorrent to help us resolve it.
+title: ''
+labels: ''
+assignees: ''
+---
+
+# Bug report
+
+<!--
+###############################################################################
+  WARNING!
+  IGNORING THE FOLLOWING TEMPLATE WILL RESULT IN THE ISSUE BEING CLOSED
+  AS INCOMPLETE/INVALID
+###############################################################################
+-->
+
+## Checklist
+
+<!--
+################################## IMPORTANT ##################################
+
+As you read and fulfill each of the following requirements below,
+put an "x" (without the quotes) between the square brackets.
+
+When doing so, ensure that no space is left between the square brackets and the "x", like so: [x]
+-->
+
+- [ ] I have read the **issue reporting guidelines** in the [contributing guidelines](https://github.com/qbittorrent/qBittorrent/blob/master/CONTRIBUTING.md), so I know how to submit a good bug report with the required information
+- [ ] I have verified that the **issue is not fixed and is reproducible** in the **[latest version](https://www.qbittorrent.org/download.php)**
+- [ ] (optional but recommended) I have verified that the **issue is not fixed and is reproducible** in the **[latest CI build](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci.yaml?query=branch%3Amaster+event%3Apush)**
+- [ ] I have **checked the [frequent/common issues list](https://github.com/qbittorrent/qBittorrent/projects/2)** and **searched** the issue tracker for similar bug reports (including closed ones) **to avoid posting a duplicate**
+- [ ] This is **not a support issue or question** better suited for either the [discussions section](https://github.com/qbittorrent/qBittorrent/discussions), [forum](https://qbforums.shiki.hu/), or [subreddit](https://www.reddit.com/r/qBittorrent/), and the [wiki](https://github.com/qbittorrent/qBittorrent/wiki) did not contain a suitable solution either
+- [ ] I have **pasted/attached the settings file and relevant log(s)** (at least the most recent one) in the **Attachments** section at the bottom of the report and filled in all the other required information below
```

Comment:
Why does this need verification? "mandatory" is already written in the corresponding section.

Reply:
Fixed.